hs_file_types/views.py

```diff
@@ -466,9 +466,17 @@ def update_metadata_element(request, hs_file_type, file_type_id, element_name,
                               'metadata_status': metadata_status,
                               'logical_file_type': logical_file.type_name()
                               }
+
+        if logical_file.type_name() == "NetCDFLogicalFile":
+            logical_file.metadata.is_update_file = True
+            logical_file.metadata.save()
+
         if logical_file.type_name() == "TimeSeriesLogicalFile":
-            ajax_response_data['is_dirty'] = logical_file.metadata.is_dirty
+            logical_file.metadata.is_update_file = True
+            logical_file.metadata.save()
+            ajax_response_data['is_update_file'] = logical_file.metadata.is_update_file
```

Comment:
Looks like ajax_response_data needs to be set for NetCDFLogicalFile as well. If true, two conditional checks above can be combined into one by checking type_name is NetCDF or TimeSeries logical file.

Reply:
Fixed